docs/secrets.md

```diff
@@ -166,6 +166,12 @@ Still, a user will be able to view the secrets using the following methods:
 To maintain the confidentiality of secret values, these operations must be strictly limited across the system by using 
 k8s RBAC and ensuring that logging into the k8s nodes as a user with elevated permissions is restricted. 
 
+##### Accessing secrets in nuclio functions
+
+The k8s secrets are passed to nuclio functions as environment variables, and their values can be retrieved directly 
+from the environment variable of the same name. For example, to access the `aws_key` secret in a nuclio function use:
+`aws_key = os.environ.get("aws_key")`
```

Comment:
````suggestion
```python
aws_key = os.environ.get("aws_key")
```
````
Better to use a code block here.